src/main/kotlin/no/nav/syfo/dialogmote/api/domain/NewDialogmoteDTO.kt

```diff
@@ -15,13 +15,31 @@ data class NewDialogmoteDTO(
 data class NewDialogmotedeltakerArbeidstakerDTO(
     val personIdent: String,
     val fritekstInnkalling: String?,
+    val innkalling: List<DocumentComponentDTO>,
 )
 
 data class NewDialogmotedeltakerArbeidsgiverDTO(
     val virksomhetsnummer: String,
     val fritekstInnkalling: String?,
+    val innkalling: List<DocumentComponentDTO>,
 )
 
+data class DocumentComponentDTO(
+    val type: DocumentComponentType,
+    val data: String,
+)
+
+enum class DocumentComponentType {
+    DOCUMENT_COMPONENT_TITLE,
+    DOCUMENT_COMPONENT_HEADER,
+    DOCUMENT_COMPONENT_SUBHEADER,
+    DOCUMENT_COMPONENT_TIME_TITLE,
+    DOCUMENT_COMPONENT_PLACE_TITLE,
+    DOCUMENT_COMPONENT_PARAGRAPH,
+    DOCUMENT_COMPONENT_LINK,
+    DOCUMENT_COMPONENT_VIDEOLINK,
+}
```

Comment:
- Jeg er enig i at fritekst og standardtekst er det samme i denne sammenhengen 👍 
- Jeg tror ikke vi trenger både `TITLE` og `HEADER`, kanskje ingen. En generisk overskrift vil man uansett ha øverst på siden, og trenger ikke være med i dokumentet. Men kanskje man trenger en for "Dialogmøte holdt dato" i referatet?
- Vi trenger vel ikke en egen `TIME_TITLE` og `PALCE_TITLE`? De kan generaliseres som subheader e.l. I min POC hadde jeg med "tittel" som et felt ved siden av type og data, slik at hvis man f.eks. sender en dato, kan man ta med tittel "møtested". Da legger vi på den tittelen over data-teksten i de tilfellene den ikke er null. Da slipper vi også å tenke på at en eventuell "time_title" må ligge rett før dato-objektet i listen, siden de alltid hører sammen.
- Trenger vi forskjellige typer `LINK`?

Reply:
Trenger alle å hete `DOCUMENT_COMPONENT_`? De ligger i en enum som heter `DocumentComponentType` (eller tilsvarende hvis vi finner et annet navn), er ikke det godt nok?

Reply:
Hvis enum'en skal brukes i Kotlin-koden synes jeg det er en fordel om enum-typen er gitt av navnene. Men i dette tilfellet er det antakelig frontend (og pdfgen) som skal forholde seg mest til disse. Så tenker at det er frontend som kan styre hvilke vi trenger og hva de skal hete. 

Hvor detaljerte vi skal dele opp disse typene (feks skille mellom link og videolink) er de "semantiske" behovene i frontend som bør styre.

Reply:
Enig i at man ikke trenger å ha DOCUMENT_COMPONENT i enum-verdiene, det er gitt av enum-klassen. Man skriver vel gjerne f.eks. DocumentComponentType.TITLE uansett når man bruker de i Kotlin-koden (hvis man ikke gjør static import av hele enum-klassen)? For frontend tenker jeg det er greit i hvert fall, Typescript-enums blir ganske likt som dette :)

Reply:
Ja, det var det med static import jeg tenkte på i det generelle tilfellet, men i dette tilfellet er det ingen grunn til å ha med enum-typen i navnene da.

Når jeg har tatt bort de John Martin mener er unødvendig er det bare PARAGRAPH og LINK igjen (hvis vi legger til et title-felt i DocumentComponentType).